### ISSUE TITLE
Final Report Peer Review

### ISSUE THREAD

**Issue by louis-stp** [Ericland/ORIE-4741-Project#11]
This project is about predicting the price of used car sales based on a very large Kaggle dataset of recent used car sales. There are many relevant features in the dataset, enough for I think an experienced appraiser to make a fair evaluation, missing perhaps photos only. There are also a lot of missing values, so many such that just dropping rows would eliminate the dataset, so other approaches would be needed. In summary, this project applied different regression and tree methods to estimate sale price, and selected random forest as the best.

Pros:
This project was very clearly written and presented. At no point was I ever unsure of what they were trying to accomplish, and how they were approaching the problem.
The steps and approaches this project uses are very logical and clear.
The results reported for each method appears logical given the properties of the methods (example: ridge regression overfits less than linear regression.) There are other examples as well, but no glaring errors or red flags were noticed.

Cons:
Why no validation set? By selecting your method (random forest) based on the results of your testing set, that means you have fitted something based on your testing set, and then that means your testing set is no longer really a testing set. You should have done everything on a validation set, made a decision, and then tested it at the end.
I disagree with the conclusion that a RMSE of 3000 is very good for this application. There is nothing wrong with saying your model is not good, making a pricing model is very difficult, and the steps you took were logical. But to say that it is very good is erroneous I think. Would you feel comfortable relying on this model to make a purchase, or to price a car that you are selling? 3000 dollars error for a 10,000 sale is not insignificant.
Lastly, I disagree with your statement that there are no obvious fairness issues. Location is a very clear proxy for race and income. According to a basic model trained with location, the same car would sell for a lower price in certain neighborhoods and for a higher price in other neighborhoods. Is this fair to those who live in the neighborhoods with lower average cost of sale? That their car would sell for less for no other reason than where they live, which is heavily associate with their race or income status?
